<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MechanismDetail</class>
 <widget class="QWidget" name="MechanismDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MechanismDetail</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1158</y>
        <width>1086</width>
        <height>3000</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>3000</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>公司概况</string>
         </property>
         <widget class="QLabel" name="label_name_logo_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>201</width>
            <height>191</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>LOGO HEER</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_team_name_11">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>70</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_name_22">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>70</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>受理金融机构编号：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_name_23">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>140</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>金融机构名称：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_team_name_12">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>140</y>
            <width>411</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>270</y>
            <width>1101</width>
            <height>20</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>1091</width>
            <height>1061</height>
           </rect>
          </property>
          <property name="title">
           <string>基本信息</string>
          </property>
          <widget class="QLineEdit" name="lineEdit_team_name_13">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>690</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_14">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>850</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_24">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>690</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>注册人：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_15">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>930</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_25">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>30</y>
             <width>131</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>公司介绍：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_26">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>620</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>行业类型：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_27">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>850</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>电话：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_16">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>1010</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_28">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>530</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>详细地址：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_17">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>530</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_18">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>620</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLineEdit" name="lineEdit_team_name_19">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>770</y>
             <width>411</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_29">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>930</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>邮箱：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="plainTextEdit_3">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>80</y>
             <width>741</width>
             <height>421</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_30">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>770</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>身份证;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_name_31">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>1010</y>
             <width>111</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>状态：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>1370</y>
            <width>1081</width>
            <height>691</height>
           </rect>
          </property>
          <property name="title">
           <string>证书</string>
          </property>
          <widget class="QLabel" name="label_name_32">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>161</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>金融机构代码证：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>70</y>
             <width>821</width>
             <height>581</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string>证件</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit_4">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>2110</y>
            <width>841</width>
            <height>421</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 100);</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_name_34">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>2060</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>备注：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>430</x>
            <y>2580</y>
            <width>181</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>保存信息</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
